src/asynchro.rs

```diff
@@ -138,6 +144,12 @@ pub struct SincFixedIn<T> {
 ///
 /// The resampling is done by creating a number of intermediate points (defined by oversampling_factor)
 /// by sinc interpolation. The new samples are then calculated by interpolating between these points.
+/// 
+/// The resampling ratio can be freely adjusted. The allowed range is -50% to +100%.
```

Comment:
This range confused me at first. +100% could be misread to mean that it doesn't go up at all! I think it would be more intuitive to say it as 50% to 200% of original rate.

Reply:
Yes you are probably right, I wasn't quite sure which one was more clear.